cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -203,6 +203,7 @@ impl LockedPackageCatalog {
     ) -> Self {
         // unpack package to avoid missing new fields
         let catalog::PackageResolutionInfo {
+            catalog: _,
```

Comment:
blocking: I think it's worth doing a `just gen-data -f` to validate this change (plus we said we'd do that whenever we did spec bumps)

Reply:
Done.  Thank you @mkenigs